Libraries/Renderer/src/renderers/native/NativeMethodsMixinUtils.js

```diff
@@ -57,12 +57,24 @@ export interface NativeMethodsInterface {
  */
 function mountSafeCallback(context: any, callback: ?Function): any {
   return function() {
-    if (
-      !callback ||
-      (typeof context.isMounted === 'function' && !context.isMounted())
-    ) {
+    if (!callback) {
       return undefined;
     }
+    if (typeof context.__isMounted === 'boolean') {
+      // TODO(gaearon): this is gross and should be removed.
+      // It is currently necessary because View uses createClass,
```

Comment:
Good point. I'm afraid that if I clarify now I'll cause a race of import script though (happened before) 😞 . Can we go with this for now?